components/loading-box/src/example.js

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import LoadingBox from '.';
+
+export default () => <LoadingBox>LoadingBox example</LoadingBox>;
```

Comment:
The example is used to generate the screenshot and should probably have the `loading` prop